packages/shared/src/components/profile/ProfileTooltip.tsx

```diff
@@ -19,49 +20,54 @@ export interface ProfileTooltipProps extends ProfileTooltipContentProps {
 }
 
 export interface ProfileTooltipContentProps {
-  user: Author;
+  userId: Author['id'];
   data?: UserTooltipContentData;
   onMouseEnter?: () => unknown;
   onMouseLeave?: () => unknown;
 }
 
 export function ProfileTooltip({
   children,
-  user,
+  userId,
   link,
   scrollingContainer,
   tooltip = {},
-}: Omit<ProfileTooltipProps, 'user'> & {
-  user?: Partial<Author>;
-}): ReactElement {
+}: Omit<ProfileTooltipProps, 'user'>): ReactElement {
+  const [id, setId] = useState<string>();
```

Comment:
We set it in a state so that we won't have to immediately request the devcard data, until only when the user triggers the tooltip.